caravel/config.py

```diff
@@ -22,6 +24,9 @@
 # ---------------------------------------------------------
 # Caravel specific config
 # ---------------------------------------------------------
+with open(os.path.join(os.path.dirname(__file__), 'static/assets/package.json')) as package_file:
+    VERSION_STRING= json.load(package_file)['version']
+
```

Comment:
Python modules are loaded at startup time so that's fine :) Anyway missing space before the `=`